src/plugins/media/nodeviews/mediaSingle.tsx

```diff
@@ -162,6 +167,15 @@ export default class MediaSingleNode extends Component<
     this.setState({
       contextIdentifierProvider: await contextIdentifierProvider
     })
+
+    // ❗ If we're not in a Firefox context, this will not throw but return undefined
+    // This patch is necessary or images can disappear when dropping them on paragraphs
+    initFirefoxDrag(firefoxElement())
```

Comment:
Do we mount a mediaSingle component for each image? 

My underlying question is "Do we attach multiple dragStart / dragEnd event to the document?".

Reply:
As it appears in my browser there is indeed only one single MediaSingle managing every MediaNode, so only one dragstart/dragend are created for this patch